LiskHQ/lisk-mobile
# Implement the token selection functionality in Home page - Closes #639

# What was the bug or feature?
Described in #639 

### How did I fix it?
 - Updated the `accountSignedIn` action to extra all address at once.
 - Created `transactionsReset` to remove the old transactions and reset the store to the initial state.
 - Fetch and store the transactions once the token is changed.
 - Fetch and store the account summary once the token is changed.
 - Create the slider in account summary.
 - Isolate the animations from the data fetching and view re-rendering to prevent glitches.
 - Reverts back to LSK if the user disables the active token.

*Extra*
 - Incognito mode icon is now always visible on the home page.

*Notes*
 - This PR doesn't implement the native gesture handler #469
 - The BTC icon is not centralized because of an issue in the icon. We can add new assets later on when they're ready.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Make sure you can change the tokens in home page by sliding to left and right.
the transactions and account summary must get updated accordingly.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented on my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
